utils/constants.ts

```diff
@@ -16,6 +18,17 @@ const constants = {
   NAME: 'name',
   NAME_PLACEHOLDER: 'Enter your name...',
   REGISTRATION_SUCCESS: 'You have successfully registered!',
+  GENDER: 'Gender',
+  AGE: 'Age',
+  WEIGHT: 'Weight (kg)',
+  HEIGHT: 'Height (cm)',
+  ABOUT_YOU: 'What are your body measurements?',
+  ABOUT_YOU_DESC: 'FILL OUT YOUR DATA',
+  FITNESS_LEVEL: 'What is your level of fitness?',
+  BEGINNER: 'Beginner',
+  INTERMEDIATE: 'Intermediate',
+  ADVANCED: 'Advanced',
+  USERS_PATH: 'users',
 };
 
```

Comment:
Consider splitting this file into modules or usages. It would be easier for maintenance.

Ex:

```tsx
const common = {
  NAME: 'Name'
}

const auth = {
  EMAIL: 'Email'
}

const wizard = {
  AGE: 'Age'
}

export {
  common,
  auth,
  wizard
}

```